tests/widgets_paragraph.rs

```diff
@@ -141,6 +141,32 @@ fn widgets_paragraph_renders_mixed_width_graphemes() {
     terminal.backend().assert_buffer(&expected);
 }
 
+#[test]
+fn white_space_wrap(){
```

Comment:
```suggestion
fn widgets_paragraph_can_wrap_with_a_trailing_nbsp() {
```
to be consistent with other tests and quickly launch all tests linked to a widget :)